src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -680,7 +687,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         elementNames = ArrayBuilder<string>.GetInstance(arguments.Count);
                         for (int j = 0; j < i; j++)
                         {
-                            elementNames.Add(name ?? "Item" + j);
+                            elementNames.Add(name ?? TupleTypeSymbol.UnderlyingMemberName(j));
```

Comment:
@VSadov, I'm not sure how to hit this with a test. If I do `(1, b: 2)` I will hit this code, but the compilation will fail with an error anyways, so the name that we put on `1` won't be visible...